postgres-appliance/scripts/configure_spilo.py

```diff
@@ -385,7 +385,7 @@ def get_placeholders(provider):
     placeholders.setdefault('CLONE_TARGET_TIME', '')
     placeholders.setdefault('CLONE_TARGET_INCLUSIVE', True)
 
-    placeholders.setdefault('LOG_SHIP_SCHEDULE', '1 0 * * *')
+    placeholders.setdefault('LOG_SHIP_SCHEDULE', '00 02 * * *')
```

Comment:
The old value looks correct for me, we want to push logs right after its rotated at midnight.